tests/core/evaluation/test_marker_tracker_loader.py

```diff
@@ -20,6 +21,36 @@ def marker_trackerstore() -> TrackerStore:
     return store
 
 
+@pytest.fixture
+def multi_session_tracker() -> TrackerStore:
+    """Sets up a TrackerStore with 1 tracker with 2 sessions."""
+    temp_db = "temp.db"
+    domain = Domain.empty()
+    store = SQLTrackerStore(domain, db=temp_db)
+    tracker = DialogueStateTracker("test123", None)
+    tracker.update_with_events(
+        [
+            UserUttered("0"),
+            UserUttered("1"),
+            SessionStarted(),
+            UserUttered("2"),
+            UserUttered("3"),
+        ],
+        domain,
+    )
+    store.save(tracker)
+    yield store
+    os.remove(temp_db)
```

Comment:
If this test is interrupted, the `temp.db` which is created in the current directory won't be removed. For this reason, other tests use the `tmp_path` fixture which will take care that the temporary file does not appear in the current directory  and that it is removed later